src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -35,13 +35,17 @@ final case class RequestSession[F[_]: Monad](
         .readTimeout(90.seconds)
     ).send()(sttpBackend, implicitly)
 
-  def sendCdf[R](r: RequestT[Empty, String, Nothing] => RequestT[Id, R, Nothing]): F[R] =
+  def sendCdf[R](
+      r: RequestT[Empty, String, Nothing] => RequestT[Id, R, Nothing],
+      contentType: String = "json",
```

Comment:
Make it `application/json`, otherwise it's confusing to set `contentType` to one thing but observe requests where it's set to `applicatin/thing`.